src/main/java/spireMapOverhaul/zones/volatileGrounds/VolatileGrounds.java

```diff
@@ -0,0 +1,79 @@
+package spireMapOverhaul.zones.volatileGrounds;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.MonsterGroup;
+import com.megacrit.cardcrawl.monsters.beyond.Exploder;
+import com.megacrit.cardcrawl.monsters.exordium.LouseNormal;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.zoneInterfaces.EncounterModifyingZone;
+import spireMapOverhaul.zones.volatileGrounds.monsters.Eruptor;
+import spireMapOverhaul.zones.volatileGrounds.monsters.HeatBlister;
+import spireMapOverhaul.zones.volatileGrounds.monsters.SunStoneShard;
+import spireMapOverhaul.zones.volatileGrounds.monsters.UnstableSunstone;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class VolatileGrounds extends AbstractZone implements EncounterModifyingZone {
+    private static final String SUNSTONE_ELITE = SpireAnniversary6Mod.makeID("SUNSTONE_ELITE");
+    private static final String EXPLOSIVE_SHAPES = SpireAnniversary6Mod.makeID("EXPLOSIVE_SHAPES");
+    private static final String HEAT_BLISTER = SpireAnniversary6Mod.makeID("HEAT_BLISTER");
+    
+    public VolatileGrounds() {
+        super("VolatileGrounds", Icons.ELITE, Icons.MONSTER);
```

Comment:
`Icons.ELITE` doesn't exist anymore. You can just delete it, you only need `Icons.MONSTER`.